app/models/ticket.js

```diff
@@ -44,5 +44,31 @@ export default ModelBase.extend({
 
   itemTotal: computed('price', 'quantity', function() {
     return this.price * this.quantity;
+  }),
+
+  /**
+   * This attribute computes total ticket price payable after inclusion
+   * of additional taxes on the base ticket price
+   */
+  ticketPriceWithTax: computed('event', function() {
```

Comment:
If you're using `tax.isTaxIncludedInPrice`, then that's what you should be watching here.